lib/db.js

```diff
@@ -0,0 +1,150 @@
+const { fetch } = require('fetch-ponyfill')();
+const FormData = require('form-data');
+const qs = require('querystring');
+const flat = require('flat');
+
+const DB_OPTIONS = require('./dbopts');
+const { httpMessage, httpBody } = require('./response-transforms');
+
+const dbOptions = (conn, options, body) => {
```

Comment:
What do you think of destructuring `options` here and elsewhere, e.g. `(conn, { method, headers }, body) => {`?